app/src/pages/common/settingsPage/ruleList/movableRuleList.jsx

```diff
@@ -27,7 +27,7 @@ const cx = classNames.bind(styles);
 const ReactGridLayout = WidthProvider(RGL);
 
 const calculateLayout = (data) =>
-  data.map((item) => ({ x: 0, y: item.order, w: 1, h: 2, i: String(item.id) }));
+  data.map((item) => ({ x: 0, y: item.order, w: 1, h: 1, i: String(item.id) }));
```

Comment:
Also please revert [this](https://github.com/reportportal/service-ui/commit/e17736943a556e100972fc6f1c45d1e4bfa072c7) commit to avoid unnecessary calculations.